### PR TITLE
fixed   Exception SNlM0e value not found in response. Check __Secure-1PSID value

### PR DESCRIPTION
**Pull request by canxin121**
added auth __Secure-1PSIDTS

#68

### CODE REVIEW

**Review by acheong08 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -21,13 +21,14 @@ usage: Bard.py [-h] --session SESSION
 
 options:
   -h, --help         show this help message and exit
-  --session SESSION  __Secure-1PSID cookie.
+  --__Secure_1PSID --__Secure_1PSIDTS       pass two cookies
```
[the remark this answers is not in the document]
Comment (canxin121) [reply README.md:24]: Of course, you can modify it, actually I forgot to consider command line using

[the remark this answers is not in the document]
Comment (amazingjoe) [reply README.md:24]: How do I make it work when my code looks like this:

chatbot = Chatbot(token)

I am passing a string of the token I grabbed from dev console.

**Review by acheong08 — APPROVED** [review]

File: src/Bard.py [hunk 2]
```diff
@@ -310,14 +317,14 @@ async def ask(self, message: str) -> dict:
         sys.exit(0)
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "--session",
-        help="__Secure-1PSID cookie.",
+        "--session --session_ts",
+        help="__Secure-1PSID cookie and __Secure_1PSIDTS cookie.",
```
Comment (acheong08) [inline src/Bard.py:321]: You removed the --session argument? Does this work?

Comment (canxin121) [reply src/Bard.py:321]: done
image

image

Comment (canxin121) [reply src/Bard.py:321]: #71

**Pull request merged**